tutorials/2d/2d_movement.rst

```diff
@@ -153,10 +153,10 @@ while up/down moves it forward or backward in whatever direction it's facing.
 
     public class Movement : KinematicBody2D
     {
-        [Export] public int Speed = 200;
+        [Export] public int speed = 200;
         [Export] public float RotationSpeed = 1.5f;
 
-        Vector2 velocity = new Vector2();
+        public Vector2 velocity = new Vector2();
         int rotationDir = 0;
```

Comment:
Actually this can still be improved further, `rotationDir` is missing `public` and `RotationSpeed` should be `rotationSpeed`.